### PR TITLE
Feature/bi 1786

### PR DESCRIPTION
**Pull request by cvacalares**
# Description
Add test for BI-1786.

# Dependencies
None

# Testing
[_Please include a link to a successful run of TAF for this change_](https://github.com/Breeding-Insight/taf/actions/runs/6138863020)

# Checklist:

- [x ] I have performed a self-review of my own code
- [ ] I have tested my code and ensured it meets the acceptance criteria of the story
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to documentation

### CODE REVIEW

**Review by dmeidlin — APPROVED** [review]

File: src/step_definitions/programManagementSteps.js [hunk 1]
```diff
@@ -775,10 +777,12 @@ When(
...
       programName = program.Name;
     } else programName = args1;
-    await page.click({
+    selector = {
       selector: `//*[@id='app']//main//a[normalize-space(.)='${programName}']`,
       locateStrategy: "xpath",
-    });
+    };
+    await page.waitForElementVisible(selector);
+    await page.click(selector);
```
Comment (mlm483) [inline src/step_definitions/programManagementSteps.js:785]: If this pattern makes TAF runs more reliable/deterministic, we should use it everywhere going forward.

File: src/features/LoginByRoleMemberTests.feature [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
 Feature: Logging with Member
 
+	@BI-822
 	Scenario: Logging in as a member of one program
 		Given user logs in as "Cucumber Member"
+		When user selects "Snacks" on program-selection page
 		Then user can see Welcome page of program
 		Then user can see "Snacks" in the upper right corner
-		And user cannot see Program Selection combo box
+		And user can see Program Selection combo box
```
Comment (mlm483) [inline src/features/LoginByRoleMemberTests.feature:10]: Why did this change?

File: src/features/LoginByRoleMemberTests.feature [hunk 3]
```diff
@@ -1,10 +1,13 @@
+@debug
 Feature: Logging with Member
 
+	@BI-822
 	Scenario: Logging in as a member of one program
 		Given user logs in as "Cucumber Member"
+		When user selects "Snacks" on program-selection page
```
Comment (mlm483) [inline src/features/LoginByRoleMemberTests.feature:7]: Why did this change?

**Review by mlm483 — APPROVED** [review]
Looks good, just have a couple questions on things I was unsure about, see comments.

**Pull request merged**